projects/step-frontend/src/lib/modules/grid/components/status-summary/status-summary.component.scss

```diff
@@ -0,0 +1,19 @@
+.token-status-IN_USE {
```

Comment:
As it was done for status-distribution, I think it will be good to put all colors in variables.scss and group them in map (See status-distribution.component.scss for example)